mountpoint-s3/src/prefetch/backpressure_controller.rs

```diff
@@ -8,7 +8,7 @@ use super::PrefetchReadError;
 #[derive(Debug)]
 pub enum BackpressureFeedbackEvent {
     /// An event where data with a certain length has been read out of the stream
-    DataRead(usize),
+    DataRead(u64, usize),
```

Comment:
nit: let's be explicit with names and/or document
```suggestion
    DataRead { offset: u64, length: usize },
```